version.php

```diff
@@ -1,4 +1,18 @@
 <?php
+/**
+ * Software version identification.
+ *
+ * @link http://open-emr.org/wiki
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @author Robert Down <robertdown@live.com>
+ */
+
+require_once __DIR__ . "/vendor/autoload.php";
```

Comment:
`.env` files are only for developmental work, you wouldn't want them in production. For production you'd want to set these variables on the server. With this functionality you are essentially saying "This ENTIRE site is a dev environment."

You wouldn't want this in globals because it's not really a functionality _of_ openemr, it just helps us debug. Down the road we could theoretically say "Are we in dev mode? Yes, cool, let's dump a bunch of debugging info to the screen" - that's a function you want to definitely *not* be able to set in the code. Dotenv files are common development processes, but shouldn't be used in production